pkg/models/settings.go

```diff
@@ -0,0 +1,36 @@
+package models
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+)
+
+type PluginSettings struct {
+	CriblOrgBaseUrl string                `json:"criblOrgBaseUrl"`
+	ClientId        string                `json:"clientId"`
+	Secrets         *SecretPluginSettings `json:"-"`
+}
+
+type SecretPluginSettings struct {
+	ClientSecret string `json:"clientSecret"`
```

Comment:
Probably a Go noob question, why ignore the field with `json:"-"` for `PluginSettings.Secrets`, but ClientSecret isn't also `json:"-"`?

Reply:
Great question...this came from Grafana's codegen.  Lemme look into this...

Reply:
@itsalvinho the `json:"-"` isn't ignored.  That's a catch-all.  And there isn't a catch-all in `SecretPluginSettings` (just the one explicitly named `clientSecret` field) since we don't expect any other fields.